packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -0,0 +1,131 @@
+import get from 'lodash/get';
+import map from 'lodash/map';
+
+import Container from '../container.class';
```

Comment:
```suggestion
import Container from '../container';
```

would be enough imo :)